node_cli/index.js

```diff
@@ -0,0 +1,59 @@
+const inquirer = require("inquirer");
+const chalk = require("chalk");
+const clear = require("clear");
+const figlet = require("figlet");
+const realmAuth = require("./realm/realmAuth");
+const main = require("./main");
+
+const output = (text, type) => {
+  if (type == "result") console.log(chalk.cyanBright(text));
+  if (type == "header") console.log(chalk.cyanBright.bold("\n" + text + "\n"));
+  if (type == "error") console.log(chalk.red.bold("\n ❗" + text + " ❗\n"));
+};
+
+let authedUser;
+
+clear();
+console.log(
+  chalk.green.bold(
+    figlet.textSync("realm-cli", {
```

Comment:
Definitely starts the app off with a bang! `realm-cli` is specifically for deploying Realm apps, so let's rename to something more Task-y.

```suggestion
    figlet.textSync("MongoDB Realm Task Tracker", {
```